src/Aspire.Dashboard/Components/Dialogs/SettingsDialog.razor

```diff
@@ -2,8 +2,13 @@
 @inject IJSRuntime JS
 @implements IDialogContentComponent
 
-<FluentRadioGroup TValue="string" Label="Theme" Value="@_currentSetting" ValueChanged="SettingChangedAsync" Orientation="Orientation.Vertical">
-    <FluentRadio Value="@ThemeSettingSystem">System</FluentRadio>
-    <FluentRadio Value="@ThemeSettingLight">Light</FluentRadio>
-    <FluentRadio Value="@ThemeSettingDark">Dark</FluentRadio>
-</FluentRadioGroup>
+<FluentStack Orientation="Orientation.Vertical" Style="display: contents
+">
+    <FluentRadioGroup TValue="string" Label="Theme" Value="@_currentSetting" ValueChanged="SettingChangedAsync" Orientation="Orientation.Vertical">
+        <FluentRadio Value="@ThemeSettingSystem">System</FluentRadio>
+        <FluentRadio Value="@ThemeSettingLight">Light</FluentRadio>
+        <FluentRadio Value="@ThemeSettingDark">Dark</FluentRadio>
+    </FluentRadioGroup>
+    <div style="height: 100%;"></div>
```

Comment:
Seems a little weird to have the extra div here just to take up the space. Could this alignment be done by setting margin-top: auto; on the version div below instead? It's not that big of a deal. Just curious

Reply:
I couldn't get it to line up nicely with margin-top. Sometimes with CSS I choose the path of least resistance...